core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -1629,139 +1629,6 @@ groups:
-    - pull-ci-openshift-openshift-azure-release-v16-images
-    - pull-ci-azure-release-v17-e2e-applysecurityupdates
-    - pull-ci-azure-release-v17-e2e-azure-conformance
-    - pull-ci-azure-release-v17-e2e-changeloglevel
-    - pull-ci-azure-release-v17-e2e-create
-    - pull-ci-azure-release-v17-e2e-etcdbackuprecovery
-    - pull-ci-azure-release-v17-e2e-forceupdate
-    - pull-ci-azure-release-v17-e2e-keyrotation
-    - pull-ci-azure-release-v17-e2e-reimagevm
-    - pull-ci-azure-release-v17-e2e-scaleupdown
-    - pull-ci-azure-release-v17-unit
-    - pull-ci-azure-release-v17-upgrade-private-v15.0
-    - pull-ci-azure-release-v17-upgrade-private-v16.0
-    - pull-ci-azure-release-v17-upgrade-private-v16.1
-    - pull-ci-azure-release-v17-upgrade-private-v17.0
-    - pull-ci-azure-release-v17-upgrade-v15.0
-    - pull-ci-azure-release-v17-upgrade-v16.0
-    - pull-ci-azure-release-v17-upgrade-v16.1
-    - pull-ci-azure-release-v17-upgrade-v17.0
-    - pull-ci-openshift-openshift-azure-release-v17-images
-    - pull-ci-azure-release-v19-e2e-applysecurityupdates
-    - pull-ci-azure-release-v19-e2e-azure-conformance
-    - pull-ci-azure-release-v19-e2e-changeloglevel
-    - pull-ci-azure-release-v19-e2e-create
-    - pull-ci-azure-release-v19-e2e-etcdbackuprecovery
-    - pull-ci-azure-release-v19-e2e-forceupdate
-    - pull-ci-azure-release-v19-e2e-keyrotation
-    - pull-ci-azure-release-v19-e2e-reimagevm
-    - pull-ci-azure-release-v19-e2e-scaleupdown
-    - pull-ci-azure-release-v19-unit
-    - pull-ci-azure-release-v19-upgrade-private-v15.0
-    - pull-ci-azure-release-v19-upgrade-private-v16.1
-    - pull-ci-azure-release-v19-upgrade-private-v17.0
-    - pull-ci-azure-release-v19-upgrade-private-v19.0
-    - pull-ci-azure-release-v19-upgrade-v15.0
-    - pull-ci-azure-release-v19-upgrade-v16.1
-    - pull-ci-azure-release-v19-upgrade-v17.0
-    - pull-ci-azure-release-v19-upgrade-v19.0
-    - pull-ci-openshift-openshift-azure-release-v19-images
-    - pull-ci-azure-release-v20-e2e-applysecurityupdates
-    - pull-ci-azure-release-v20-e2e-azure-conformance
-    - pull-ci-azure-release-v20-e2e-changeloglevel
-    - pull-ci-azure-release-v20-e2e-create
-    - pull-ci-azure-release-v20-e2e-etcdbackuprecovery
-    - pull-ci-azure-release-v20-e2e-forceupdate
-    - pull-ci-azure-release-v20-e2e-keyrotation
-    - pull-ci-azure-release-v20-e2e-reimagevm
-    - pull-ci-azure-release-v20-e2e-scaleupdown
-    - pull-ci-azure-release-v20-unit
-    - pull-ci-azure-release-v20-upgrade-private-v15.0
-    - pull-ci-azure-release-v20-upgrade-private-v16.1
-    - pull-ci-azure-release-v20-upgrade-private-v17.0
-    - pull-ci-azure-release-v20-upgrade-private-v19.0
-    - pull-ci-azure-release-v20-upgrade-v15.0
-    - pull-ci-azure-release-v20-upgrade-v16.1
-    - pull-ci-azure-release-v20-upgrade-v17.0
-    - pull-ci-azure-release-v20-upgrade-v19.0
-    - branch-ci-openshift-release-master-release-controller-annotate
```

Comment:
From this job, they are not azure jobs, right?

Reply:
rebase error. will fix